tests/p2p/test_kademlia.py

```diff
@@ -0,0 +1,322 @@
+import asyncio
```

Comment:
Did you really intend to duplicate this test here? I'm keeping all the p2p stuff under evm/p2p to make it easier to eventually split it into its own repo.  Either way, you should at least remove this one and rename the original